presto-docs/src/main/sphinx/conf.py

```diff
@@ -81,6 +81,8 @@ def get_version():
 
 default_role = 'backquote'
 
+copybutton_image_path = 'presto-copy-button.svg'
```

Comment:
i would rename the file to use just `copy-button.svg`

Reply:
Cannot do that - I need different name than the file provided by the plugin.

Reply:
And I assume you cant use that one because it looks bad?

Reply:
I've changed the name - the modified icon has a thicker line so it's more visible